install_config/aggregate_logging.adoc

```diff
@@ -250,25 +255,21 @@ operations logs.
 |Variables for the operations log cluster.
 |===
 +
+[NOTE]
 When setting the `*ENABLE_OPS_CLUSTER*` parameter to `*true*`, Fluentd splits
 logs between the Elasticsearch cluster and a cluster reserved for operations
 logs. This means a second Elasticsearch and Kibana are deployed. The deployments
 are distinguishable by the *-ops* included in their names.
 +
-This creates a deployer pod and prints its name. Wait until the pod
-is running; this can take up to a few minutes to retrieve the deployer
-image from its registry. You can watch it with:
+Running the deployer creates a deployer pod and prints its name. Wait until the pod is running to retrieve the deployer image from its registry. You can watch it with:
```

Comment:
By shortening the second sentence here from 
"Wait until the pod is running; this can take up to a few minutes to retrieve the deployer image from its registry."
to 
"Wait until the pod is running to retrieve the deployer image from its registry."
I feel like some meaning is lost, it almost sounds like you're telling them to wait until the pod is running before trying to retrieve the deployer image from its registry.